plugin/src/main/java/spp/jetbrains/sourcemarker/status/LogStatusBar.java

```diff
@@ -120,11 +130,13 @@ public Icon getIcon() {
         }).collect(Collectors.toList());
         lookup = text -> scopeVars.stream()
                 .filter(v -> {
-                    String var = substringAfterLast(" ", text.toLowerCase());
-                    return var.startsWith("$") && !var.substring(1).equals(v) && v.toLowerCase().contains(var.substring(1));
+                    String var = substringAfterLast(SPACE, text.toLowerCase());
+
+                    return var.startsWith(DOLLOR) && !var.substring(1).equals(v) && v.toLowerCase().contains(var.substring(1)) ||
+                           var.startsWith("${") && !var.substring(2).equals(v) && v.toLowerCase().contains(var.substring(2));
```

Comment:
DOLLOR shoudl be DOLLAR and you've missed a `$` here

Reply:
Constant renamed. I don't understand the second part. I have not missed $. Please correct me

Reply:
`startsWith("${")`

Reply:
right now the code checks startswith of both $ or ${ and it works. Please check